Makefile

```diff
@@ -8,8 +8,8 @@ OCCLIBS=-lTKBRep -lTKG2d -lTKG3d -lTKGeomBase \
 ifeq "$(OCEINCLUDE)" ""
 
 #CXX=gcc-4.4
-CXXFLAGS += -I/usr/include/opencascade
-LDFLAGS += -L/usr/lib/opencas -L/usr/lib ${OCCLIBS}
+CXXFLAGS += -I/usr/local/Cellar/opencascade/7.3.0p3/include/opencascade
+LDFLAGS += -L/usr/local/Cellar/opencascade/7.3.0p3/lib -L/usr/lib ${OCCLIBS}
```

Comment:
Thanks for the effort you put in getting this to work and thanks for creating a PR. However, this is not a good solution for everyone, because this path is specific to macOS (using, I think, homebrew to install opencascade), and the version number may change as well. It would be good to find a solution that works on both linux and mac, ideally without an if conditions, but that would be a fallback. Does homebrew create any symbolic links that are more similar to how this works on linux? (i.e., `/usr/include` + `/usr/lib`)

Reply:
Totally agree! There's a fresh commit in the PR that handles this. Perhaps the README should also be updated to document the environment variables?

Reply:
awesome, thanks! Regarding the README: yes sounds good. As you can probably tell, I'm not maintaining this repo very actively -- also because it doesn't require much. But if you send a PR for that I'm happy to merge that as well.